contracts/src/TokenRemote/NativeTokenRemote.sol

```diff
@@ -70,28 +112,12 @@ contract NativeTokenRemote is INativeTokenRemote, IWrappedNativeToken, ERC20, To
     INativeMinter public constant NATIVE_MINTER =
         INativeMinter(0x0200000000000000000000000000000000000001);
 
-    /**
-     * @notice Percentage of burned transaction fees that will be rewarded to a relayer delivering
-     * the message created by calling calling reportBurnedTxFees().
-     */
-    uint256 public immutable burnedFeesReportingRewardPercentage;
-
-    /**
-     * @notice Total number of tokens minted by this contract through the native minter precompile.
-     */
-    uint256 public totalMinted;
-
-    /**
-     * @notice The balance of BURNED_TX_FEES_ADDRESS the last time burned fees were reported to the TokenHome instance.
-     */
-    uint256 public lastestBurnedFeesReported;
-
     /**
      * @dev When modifier is used, the function can only be called after the contract is fully collelateralized,
      * accounting for the initialReserveImbalance.
      */
     modifier onlyWhenCollateralized() {
-        require(isCollateralized, "NativeTokenRemote: contract undercollateralized");
+        require(getIsCollateralized(), "NativeTokenRemote: contract undercollateralized");
```

Comment:
May want to consider making this a function and having it take `NativeTokenRemoteStorage storage` as a param to avoid an extra read of the storage slot.

Reply:
Ah, nevermind I see it's part of `TokenRemote`, not `NativeTokenRemote`. Not sure if there's a clear option for trying to avoid the additional reads in those cases. Worth thinking about but not a big deal either way, definitely would favor readibility.